tests/plugins/test_check_and_set_rebuild.py

```diff
@@ -153,12 +155,16 @@ def test_check_is_not_rebuild(self, tmpdir, namespace, monkeypatch, reactor_conf
         assert workflow.prebuild_results[CheckAndSetRebuildPlugin.key] is False
         assert not is_rebuild(workflow)
 
+    @pytest.mark.parametrize('base_from_scratch', (True, False))
     @pytest.mark.parametrize('from_latest', (None, True, False))
-    def test_check_is_rebuild(self, tmpdir, monkeypatch, reactor_config_map, from_latest):
+    def test_check_is_rebuild(self, caplog, tmpdir, monkeypatch, reactor_config_map,
+                              base_from_scratch, from_latest):
         key = 'is_autorebuild'
         value = 'true'
 
-        workflow, runner = self.prepare(tmpdir, key, value, reactor_config_map=reactor_config_map)
+        workflow, runner = self.prepare(tmpdir, key, value, reactor_config_map=reactor_config_map,
+                                        base_from_scratch=base_from_scratch)
+        workflow.builder.base_from_scrath = base_from_scratch
```

Comment:
Typo: scrath -> scratch

I wonder why this didn't cause any test failures.

Reply:
yeah that should be deleted as it is set via prepare :)